### ISSUE TITLE
Support different ontology versions

### ISSUE THREAD

**Issue by urbanmatthias** [simphony/osp-core#526]
Install:
`pico install emmo@1.0.1` or `pico install emmo==1.0.1`.

**Comment by yoavnash** [comment]
Is it possible to ask pico what version is installed? At the moment, it just says `emmo`  but no information about the version appears.

**Comment by kysrpex** [comment]
> Is it possible to ask pico what version is installed? At the moment, it just says `emmo` but no information about the version appears.

No, at the moment I think it is not possible to access the ontology metadata from OSP-core.
